mlflow/pytorch/_lightning_autolog.py

```diff
@@ -286,7 +287,7 @@ def on_test_end(self, trainer, pl_module):
         self.metrics_logger.flush()
 
 
-class MlflowModelCheckpointCallback(pl.Callback, metaclass=ExceptionSafeAbstractClass):
+class MlflowModelCheckpointCallback(pl.Callback, MlflowModelCheckpointCallbackBase):
```

Comment:
Although at this time we don't expect users to directly use this class, since this is a public class, can we add a code example on how to use it?

Reply:
ohh we cannot make it a private class because we are importing it in a different file.